lib/features/onboarding/ui/pages/personal_info_page.dart

```diff
@@ -62,29 +93,32 @@ class _PersonalInfoPageState extends State<PersonalInfoPage> {
                   ),
                 ),
                 SizedBox(height: theme.sizing.height.s3),
-                const MinyTextField(
-                  hintText: OnboardingConstants.enterCityText,
+                ExpandableSearchField(
+                  allItems: cityList,
+                  onSelected: (value) {
+                    setState(() {
+                      selectedCity = value;
+                    });
+                  },
                 ),
                 SizedBox(height: theme.sizing.height.s10),
                 ..._buildChipsSection(
                   theme,
                   label: OnboardingConstants.genderLabel,
                   options: genderOptions,
                   selectedValue: selectedGender,
-                  onSelected: (value) => setState(
-                    () => selectedGender = value,
-                  ),
+                  onSelected: (value) => setState(() => selectedGender = value),
                 ),
                 SizedBox(height: theme.sizing.height.s10),
                 ..._buildChipsSection(
                   theme,
                   label: OnboardingConstants.maritalStatusLabel,
                   options: maritalOptions,
                   selectedValue: selectedMaritalStatus,
-                  onSelected: (value) => setState(
-                    () => selectedMaritalStatus = value,
-                  ),
+                  onSelected: (value) =>
```

Comment:
Format with coma (,)
```suggestion
                  onSelected: (value) => setState(() => selectedMaritalStatus = value),
```

Reply:
done